docs/source/1.0/spec/core/model.rst

```diff
@@ -366,14 +430,54 @@ semantics, unlike the very generic simple and aggregate shapes, as they
+          └───────────────┘            └─────────────┘       └─────────────┘
+                  △                     △                     △
+    ┌──────────┐  │  ┌─────────┐        │    ┌────────────┐   │    ┌─────────────────────────┐
+    │blob      │──┼──│boolean  │        ├────│    List    │   │    │         Service         │
+    └──────────┘  │  └─────────┘        │    ├────────────┤   │    ├─────────────────────────┤
+    ┌──────────┐  │  ┌─────────┐        │    │member      │   │    │version                  │
+    │document  │──┼──│string   │        │    └────────────┘   ├────│operations: [Operation]? │
+    └──────────┘  │  └─────────┘        │    ┌────────────┐   │    │resources: [Resource]?   │
+    ┌──────────┐  │  ┌─────────┐        ├────│    Set     │   │    └─────────────────────────┘
+    │byte      │──┼──│short    │        │    ├────────────┤   │    ┌─────────────────────────┐
+    └──────────┘  │  └─────────┘        │    │member      │   │    │        Operation        │
+    ┌──────────┐  │  ┌─────────┐        │    └────────────┘   │    ├─────────────────────────┤
+    │integer   │──┼──│long     │        │    ┌────────────┐   │    │input: Structure?        │
+    └──────────┘  │  └─────────┘        ├────│    Map     │   ├────│output: Structure?       │
+    ┌──────────┐  │  ┌─────────┐        │    ├────────────┤   │    │errors: [Structure]?     │
+    │float     │──┼──│double   │        │    │key         │   │    └─────────────────────────┘
+    └──────────┘  │  └─────────┘        │    │value       │   │    ┌─────────────────────────┐
+    ┌──────────┐  │  ┌─────────┐        │    └────────────┘   │    │        Resource         │
+    │bigDecimal│──┼──│timestamp│        │    ┌────────────┐   │    ├─────────────────────────┤
+    └──────────┘  │  └─────────┘        ├────│ Structure  │   │    │identifiers?             │
+    ┌──────────┐  │                     │    └────────────┘   │    │create: Operation?       │
+    │bigInteger│──┘                     │    ┌────────────┐   │    │put: Operation?          │
+    └──────────┘                        └────│   Union    │   │    │read: Operation?         │
+                                             └────────────┘   └────│update: Operation?       │
+                                                                   │delete: Operation?       │
+                                                                   │list: : Operation?       │
+                                                                   │operations: [Operation]? │
+                                                                   │collectionOperations:    │
+                                                                   │    [Operation]?         │
+                                                                   │resources: [Resource]?   │
+                                                                   └─────────────────────────┘
```

Comment:
Try and reduce some of the horizontal spacing, for me I end up having a scroll bar to see the far right-hand side